processes/consumer/flush.go

```diff
@@ -90,6 +90,7 @@ func Flush(ctx context.Context, inMemDB *models.DatabaseData, dest destination.B
 
 			if err != nil {
 				tags["what"] = "merge_fail"
+				tags["retryable"] = fmt.Sprintf("%t", dest.IsRetryableError(err))
```

Comment:
Do fmt.Sprint(dest.IsRetryableError(err)) so you don't need to worry about string interpolation?

Reply:
Good call.